src/screens/ChatScreen/components/ChatListItem.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { UserPhoto } from 'app/components';
+import { FirebaseDocument } from 'app/types/firebase';
+
+interface Props {
+  item: FirebaseDocument;
+}
+
+const styles = StyleSheet.create({
+  listItem: {
+    flex: 1,
+    height: 150,
```

Comment:
in general it's better to style using paddings and leave `height` as undefined. It will scale better when the user has custom font size settings from the accessibility options